Cargo.toml

```diff
@@ -23,10 +23,10 @@ maintenance = { status = "as-is" }
 nightly = []
 
 [dependencies]
-ndarray = "0.13"
+ndarray = "0.15.6"
 num-traits = "0.2"
 log = { version = "0.4", features = ["max_level_debug", "release_max_level_warn"] }
 
 [dev-dependencies]
-rand = "0.7"
-rustc_version = "0.2"
+rand = "0.8.5"
+rustc_version = "0.4.0"
```

Comment:
You do not need to specify the minor version, this way the crates that uses this will be able to update the version on their own, so e.g.
```
rand = "0.8"
```
would be enough